.github/renovate.json5

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":automergeMinor",
+    ":automergeDigest",
+    ":labels(renovate,renovate/{{depName}})",
+    "helpers:pinGitHubActionDigests",
+  ],
+  "regexManagers": [
+    {
+      "description": "Update actions/runner version",
+      "fileMatch": ["^\\.github/workflows/build\\.yaml$"],
+      "matchStrings": ["RUNNER_VERSION=(?<currentValue>.*?)\\s"],
+      "depNameTemplate": "actions/runner",
+      "datasourceTemplate": "github-releases",
```

```suggestion
      "datasourceTemplate": "github-releases",
      "extractVersionTemplate": "^v(?<version>.*)$"
```

Since `actions/runner` releases are prefixed by `v`, should we remove that?

- https://github.com/actions/runner/releases
- https://zenn.dev/wim/articles/renovate_no_settei_wo_shirabetayo#extractversiontemplate

And before we confirmed it working, can we disable auto-merge for a while?

---

> And before we confirmed it working, how about disabling auto-merge for a while?
+1 for this comment!

---

Nice catch! I will disable auto-merge of actions/runner.

---

I added https://github.com/quipper/actions-runner/pull/1/commits/f6b61fab491812b6661ce84c9d5773b38800ec05